docs/checkit.js

```diff
@@ -460,8 +460,8 @@ CheckIt.prototype.generateCheckmarks = function(calObj, $calendarDiv) {
 
 CheckIt.prototype.removeEmptyWeeks = function(calObj, $calendarDiv) {
     // Remove empty weeks from the calendar
-        
-    $calendarDiv.find('.month > .week').each( function(index) {
+    
+    $calendarDiv.find('.month > tbody > .week').each( function(index) {
```

Comment:
Just use `'.month .week'` probably. You don't want this to break just because you change how the calendar part is structured in html